OpenRA.Mods.Common/UpdateRules/Rules/20221203/ExplicitSequenceFilenames.cs

```diff
@@ -315,8 +326,6 @@ public override IEnumerable<string> UpdateSequenceNode(ModData modData, MiniYaml
 			foreach (var sequenceNode in imageNode.Value.Nodes.ToList())
 				if (implicitInheritedSequences.Contains(sequenceNode.Key) && !sequenceNode.Value.Nodes.Any())
 					imageNode.RemoveNode(sequenceNode);
-
-			yield break;
```

Comment:
Every update rule yields break. Why are you removing it

Reply:
It's at the end of the method, so unnecessary:
![image](https://user-images.githubusercontent.com/7704140/219509657-21b3c749-5780-4a76-80e5-99f68a3b2d79.png)

Reply:
Does it say that for all update rules?

Reply:
In fact it does, but I'm not sure if it's worth touching every update rule over it. I can revert this change if you think it is too out of place.

Reply:
This rule doesn't exist in https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/

It seems this is something picked up by SonarSource. I'd rather you revert the change here, and we then do a follow up PR where we remove all redundant jumps

Reply:
Reverted.